Bổ sung thêm phần đánh dấu chưa xử lý

tuanpb1988 opened the pull request:
* Khi chưa xử lý: Thì có nút "Đánh dấu đã xử lý"
* Khi đã xử lý" Thì có nút "Đánh dấu chưa xử lý"
* Khi đánh dấu chưa đọc thì đồng thời đánh dấu là "Chưa xử lý" luôn

Commit:
Block robots in some areas
- Modules: users, banners (click func), comment, freecontent, menu, statistics, two-step-verification
- Admin CP

vuthao committed:
Merge pull request #3035 from hoaquynhtim99/nukeviet4.4

Update default domains whitelist (fix#3019)

Commit:
Create CHANGE.md

Ghi chú các thay đổi CSDL

Commit:
Create processed.png

Thêm icon đã xử lý

Commit:
Thêm chức năng đánh dấu đã xử lý Email

Commit:
Update action_mysql.php của module contact

Commit:
update file lang

Commit:
Xóa thư mục VINADES, file CHANGE.md

Commit:
Sửa lỗi chức năng đánh dấu đã xử lý - module Contact

Commit:
Bổ sung thêm key cho file lang en,fr

Commit:
sửa lỗi file main.php

Commit:
xóa phần thay đổi của file .gitignore

Commit:
hiển thị 'N/A' nếu không tìm thấy ID admin thực hiện

Commit:
bổ sung key file lang admin

Commit:
sửa file view.php: thay thế tên bảng nv4_users -> NV_USERS_GLOBALTABLE

Commit:
sửa lỗi file main.php

Commit:
Đánh dấu is_read=1 khi đánh dấu đã thực hiện

Commit:
Sửa lỗi "chưa khai báo biến reply_aid" file view.php

Commit:
Bổ sung thêm phần đánh dấu chưa xử lý

* Khi chưa xử lý: Thì có nút "Đánh dấu đã xử lý"
* Khi đã xử lý" Thì có nút "Đánh dấu chưa xử lý"
* Khi đánh dấu chưa đọc thì đồng thời đánh dấu là "Chưa xử lý" luôn

Commit:
Bổ sung ghi log cho module Menu

Bổ sung ghi log cho các thao tác thêm, sửa, xóa, đình chỉ, kích hoạt, thay đổi thứ tự của menu và bộ menu.

hoaquynhtim99 commented on modules/contact/admin/main.php:
```diff
@@ -43,14 +46,27 @@
+        nv_insert_logs(NV_LANG_DATA, $module_name, $lang_module['mark_as_processed'], 'ID: ' . implode(',', $sends), $admin_info['userid']);
+    } else {
+        $mark = 0;
+        $is_read = 0;
```
@tuanpb1988 chỗ này nhầm rồi anh. Khi đánh dấu chưa xử lý thì không có nghĩa phải đánh dấu là chưa đọc. Vẫn có thể đọc rồi nhưng chưa xử lý mà. Nên khi đánh dấu chưa xử lý thì anh cứ để như hiện tại. Đã đọc thì vẫn là đã đọc mà chưa đọc thì vẫn là chưa đọc

hoaquynhtim99 commented on modules/menu/admin/change_active.php:
```diff
@@ -26,6 +26,13 @@
+}
+
+
 
```
@tuanpb1988 chỗ này thừa một dòng trống anh

hoaquynhtim99 commented on modules/menu/admin/menu.php:
```diff
@@ -29,12 +29,16 @@
         if (empty($arr['id'])) {
             $error = $lang_module['errorsave'];
+        } else {
+            nv_insert_logs(NV_LANG_DATA, $module_name, 'Add menu','Menu id: ' . $arr['id'], $admin_info['userid']);
```
Chỗ này chưa chuẩn anh `'Add menu','Menu id:` sau dấu `,` thiếu khoảng trống

hoaquynhtim99 commented on modules/menu/admin/menu.php:
```diff
@@ -29,12 +29,16 @@
         if (! $stmt->execute()) {
             $error = $lang_module['errorsave'];
+        } else {
+            nv_insert_logs(NV_LANG_DATA, $module_name, 'Edit menu','Menu id: ' . $arr['id'], $admin_info['userid']);
```
Chỗ này sau dấu phảy cũng thiếu khoảng trống

hoaquynhtim99 commented on modules/menu/admin/rows.php:
```diff
@@ -352,6 +356,7 @@
     foreach ($array_id as $id) {
         nv_menu_del_sub($id, $post['parentid']);
     }
+    nv_insert_logs(NV_LANG_DATA, $module_name, 'Del row menu', 'Row menu id: ' . implode(',',$array_id), $admin_info['userid']);
```
Chỗ này code cũng chưa chuẩn, thiếu khoảng trống sau dấu phảy

hoaquynhtim99 commented on themes/admin_default/js/contact.js:
```diff
@@ -24,6 +24,18 @@ function mark_as_unread() {
+		"error" == a.status ? alert(a.mess) : window.location.href = a.mess
+	});
+	return !1
+}
```
Đoạn này dùng tab để thụt đầu dòng, anh thay 1 tab bằng 4 space

Commit:
Sửa lỗi file main.php, module contact

Commit:
chuẩn PSR-12 file contact.js

Commit:
Sửa lỗi chưa chuẩn PSR-12, module menu

change_active.php, change_weight_row.php, del_row.php, main.php, menu.php, rows.php

Commit:
Bổ sung cột hiển thị lượt xem trong quản trị của module about
